src/core/helpers/page.ts

```diff
@@ -59,6 +59,14 @@ export function createGetServerSideProps(options: CreateGetServerSideProps) {
     }
     if (ctx.me?.user) {
       const { features } = ctx.me;
+      if (ctx.resolvedUrl === "/") {
```

Comment:
Are you sure the user has the right to access the workspaces ?

Reply:
What about the ifs a bit below this one ?

Reply:
My bad, it should come after the features flag check